command/pr_create.go

```diff
@@ -265,9 +268,9 @@ func prCreate(cmd *cobra.Command, _ []string) error {
 		didForkRepo = true
 	}
 
-	headBranchLabel := headBranch
+	headBranchRef := headBranch
```

Comment:
Let's avoid renaming variables unless there's a strong reason to. It makes the overall PR diff harder to review for us.